pkg/combustion/cleanup.go

```diff
@@ -0,0 +1,37 @@
+package combustion
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+	"go.uber.org/zap"
+)
+
+const (
+	cleanupScriptName    = "99-cleanup-combustion.sh"
+	cleanupComponentName = "cleanup"
+)
+
+//go:embed templates/99-cleanup-combustion.sh
+var cleanupScript string
+
+func configureCleanup(ctx *image.Context) ([]string, error) {
+	if ctx.ImageDefinition.Image.ImageType != "raw" {
```

Comment:
Can you compare this to the constant `TypeRAW` in `definition` instead? I prefer to avoid inline constants like this.